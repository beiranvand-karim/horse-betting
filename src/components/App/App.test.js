/* global describe, it, expect*/
import React from "react"
import App from "./"
import {MemoryRouter} from "react-router-dom"
import renderer from "react-test-renderer"
import {shallow} from "enzyme"

describe('<App/>', () => {
   it('should render without crashing', () => {
      shallow(<App/>)
   });

   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><App/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});