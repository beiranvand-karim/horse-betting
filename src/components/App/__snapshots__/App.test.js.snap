// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should matches the snapshot 1`] = `
<div>
  <h1>
    landing page
  </h1>
  <form
    action=""
    onSubmit={[Function]}
  >
    <input
      id="gamedId"
      name="gamedId"
      onChange={[Function]}
      type="text"
      value=""
    />
    <button>
      search
    </button>
  </form>
</div>
`;
